app/model/spree/refund_decorator.rb

```diff
@@ -1,6 +1,10 @@
-Spree::Refund.class_eval do
-  def pos_order_id
-    # NOTE(cab): Should we use delegates here?
-    payment.order.pos_order_id
+module Spree
+  module RefundDecorator
+    def pos_order_id
+      # NOTE(cab): Should we use delegates here?
```

Comment:
The note written there was mainly used for code-reviewer to give me feedback. It's an open question :)